spring-cloud-gcp-autoconfigure/src/test/java/com/google/cloud/spring/autoconfigure/firestore/GcpFirestoreAutoConfigurationTests.java

```diff
@@ -60,6 +60,7 @@ void testDatastoreOptionsCorrectlySet() {
         context -> {
           FirestoreOptions datastoreOptions = context.getBean(Firestore.class).getOptions();
           assertThat(datastoreOptions.getProjectId()).isEqualTo("test-project");
+          assertThat(datastoreOptions.getDatabaseId()).isEqualTo("(default)");
```

Comment:
why there's a pair of parentheses?

Reply:
That's just the name of the default database.
See: https://firebase.google.com/docs/firestore/manage-databases#access_a_named_database_with_a_client_library